django_rest/tests/test_employees.py

```diff
@@ -0,0 +1,88 @@
+import pytest
+from django.urls import reverse
+from django_rest.models import Employee
+from django.contrib.auth.models import User
+from rest_framework.test import APIClient
+
+
+@pytest.fixture
+def admin_client(api_client=APIClient()):
```

Comment:
Is there a need to provide `api_client=APIClient()` as an argument? Maybe just move it one line below to the function scope?